src/filter.ts

```diff
@@ -358,7 +358,7 @@ type CollapseL2<
     : never
         : never);
 // All queries
-type AllValidFilterQueries = InjectShortcuts;
+export type AllValidFilterQueries = InjectShortcuts;
```

Comment:
Use `FilterQuery` instead of `AllValidFilterQueries`. This helper type was introduced to prevent type alias expansion in editor tooling, and to make the API reference more readable. `FilterQuery` also has docs.